src/helpers.php

```diff
@@ -9,6 +9,20 @@ function module_path($name)
     }
 }
 
+if ( ! function_exists('config_path'))
+{
```

Comment:
Please run phpcsfixer. The brace should be on the same line.

Reply:
done